tutorials/physics/using_kinematic_body_2d.rst

```diff
@@ -272,7 +360,39 @@ And the code for the Bullet:
 
  .. code-tab:: csharp
 
-    Coming soon
+    using Godot;
+    using System;
+
+    public class Bullet : KinematicBody2D
+    { 
+        int speed = 750;
+        Vector2 velocity = new Vector2();
+
+        public void Start(Vector2 pos, float dir)
+        {
+            Rotation = dir;
+            Position = pos;
+            velocity = new Vector2(speed, 0).Rotated(Rotation);
+        }
+
+        public override void _PhysicsProcess(float delta)
+        {
+            var collsion = MoveAndCollide(velocity * delta);
+            if (collsion != null)
+            {
+                velocity = velocity.Bounce(collsion.Normal);
+                if (collsion.Collider.HasMethod("Hit"))
+                {
+                    collsion.Collider.Hit();
+                }
+            }
+        }
+
+        public void onVisibilityNotifier2DScreenExited()
```

Comment:
According to a discussion in #1365, it seems that we decided to follow PascalCase for signal callbacks. So, I believe we should treat it like any other methods regarding the naming convention, making it start with a upper case letter.